server/tests/load_postgres_data.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import getpass
+import os
+import subprocess
+
+conf = {
+    "staging": {
+        "port": 5432,
+    },
+    "production": {
+        "port": 5433,
+    },
+}
+
+
+def run_pg_cmd_on_guest(guest_cmd, password):
+    """Run a postgres command in the guest container."""
+    return subprocess.run([
+        "docker-compose exec -e PGPASSWORD=%s postgres bash -c '%s'" % (
+            password, guest_cmd),
+    ], shell=True, check=True)
+
+
+def parse_args():
+    """Parse arguments for the posgres data loader command."""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("environment", default="staging",
```

Comment:
```suggestion
    parser.add_argument("environment", default="staging", choices=conf,
```
and you'll not have to `if parsed_args.environment not in conf` below.